app/models/katello/concerns/pulp_database_unit.rb

```diff
@@ -127,7 +127,7 @@ def import_for_repository(repository)
 
       def sync_repository_associations(repository, options = {})
         additive = options.fetch(:additive, false)
-        pulp_ids = options.fetch(:pulp_ids)
+        pulp_ids = options.fetch(:pulp_ids, nil)
```

Comment:
Not asking for change here, but I like to use `.dig` in this case since it will return nil instead of raising an error